platform/engineering/design-docs/2020-04-09-separate-content-build.md

```diff
@@ -168,8 +168,8 @@ The current front end build will be split up into two distinct builds:
     - Output: JavaScript and CSS bundles
     - This essentially maps to the current full deploy minus the content
 
-The output of both these builds will be deployed to a single S3 bucket with
-safeguards in place to ensure they don't override each other.
+There will be two S3 buckets. One will hold the output of the application build (including .js and .css), and the other will hold the output of the content build (including .html and .pdf).
```

Comment:
Do you think it's worth clarifying that there will be three buckets total when we're done; the current bucket, the new application bucket, and the new content bucket?

Reply:
That's a good point.  Should it say:

> There will be two _new_ S3 buckets. ...
> The original bucket will still exist to hold teamsite assets.

Reply:
I think that makes sense.